### PR TITLE
Joey development - Changes made by the create-react-app setup command

### PR DESCRIPTION
**Pull request by Jortiz23**
This is the initial framework of the react project. This is being committed to have a common file structure across all branches and to reduce problems going forward

### CODE REVIEW

**Comment by Jortiz23** [comment]
@googlebot I signed it!

**Comment by Jortiz23** [comment]
@googlebot I signed it!

**Comment by Jortiz23** [comment]
@googlebot I signed it!

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (Snehalb26) [inline src/App.css:1]: Tags and properties inside tags should be in alphabetical order throughout the file.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (NathanBricault) [inline public/index.html:10]: Is this entire pull request the raw, unedited result of running a setup command? If so that should be stated somewhere. 

Still, seems like basic strings like this should be set (or make it clear that you'll be setting them in another pull request)

File: src/serviceWorker.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                'New content is available and will be used when all ' +
+                  'tabs for this page are closed. See https://bit.ly/CRA-PWA.'
+              );
+
+              // Execute callback
+              if (config && config.onUpdate) {
+                config.onUpdate(registration);
+              }
+            } else {
+              // At this point, everything has been precached.
```
Comment (Snehalb26) [inline src/serviceWorker.js:82]: Do we need to commit these files as it is now and you will revisit this later? Lot's of comments can be omitted here as code itself says what it is doing, but if you want to change it later based on implementation that's fine.

File: public/index.html (hunk 2 above)
Comment (Jortiz23) [reply public/index.html:10]: Yes, this whole pull request is purely from the setup command. I was unsure how to word it properly, but this is essentially the foundations that we all will be working from

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
```
Comment (shainachen) [inline public/manifest.json:20]: This seems like it's from the sample app, I think we'd probably like to modify some of this before it goes into master. The names (name, shortname) should correspond to our project name. Do we want/need to have all these logos? I imagine the react logos are for their sample app, not sure if we have to keep those.

File: src/serviceWorker.js (hunk 3 above)
Comment (Jortiz23) [reply src/serviceWorker.js:82]: Everything will be revisited later. These are just the changes made by the create-react-app setup command.

File: src/App.css (hunk 1 above)
Comment (Jortiz23) [reply src/App.css:1]: Done

File: public/manifest.json (hunk 4 above)
Comment (Jortiz23) [reply public/manifest.json]: I left it in because I was just committing the unedited changes from running the react command, but I think you're right. I got rid of the things we for sure won't need for our project.
